site/content/get-started.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Get started
+toc: true
+weight: 200
+docs: DOCS-000
+---
+
+This is a guide for getting started with NGINX Gateway Fabric. It explains how to:
+
+- Set up a [kind (Kubernetes in Docker)](https://kind.sigs.k8s.io/) cluster
+- Install [NGINX Gateway Fabric](https://blog.nginx.org/blog/5-things-to-know-about-nginx-gateway-fabric) with [NGINX](https://nginx.org/)
```

Comment:
I think it may be more appropriate to link to our Github page here. This is an older blog.